sdks/core/src/main/java/org/gbif/pipelines/core/converters/OccurrenceJsonConverter.java

```diff
@@ -38,13 +43,15 @@
 @Builder
 public class OccurrenceJsonConverter {
 
+  public static final String GBIF_BACKBONE_DATASET_KEY = "d7dddbf4-2cf0-4f39-9b2a-bb099caae36c";
```

Comment:
will this never change?

Reply:
It won't / shouldn't, but if gbif-api is in here then it could use this: https://github.com/gbif/gbif-api/blame/dev/src/main/java/org/gbif/api/model/Constants.java#L28

Reply:
This block of code *should* be a short lived thing. Once we can remove the `gbifClassification` block from the ES index (currently required by hosted portals), this code will be removed.

Reply:
changed to use gbif-api constants